app/src/main/java/com/conupods/MapsActivity.java

```diff
@@ -23,8 +30,12 @@ protected void onCreate(Bundle savedInstanceState) {
         SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
                 .findFragmentById(R.id.map);
         mapFragment.getMapAsync(this);
-    }
 
+        Button one = (Button) findViewById(R.id.SGW);
```

Comment:
Buttons should be renamed to be more representative of their function

Reply:
Done